docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -89,44 +83,58 @@ Next, you need to remove the empty users from your MySQL database:
+First, you need to make sure the interfaces name will be the same on *all servers*. See 'Setting the interfaces name on CentOS 7' in the Appendix section of this document if all your servers don't already have the same interface names.
 
 Next, you will need to configure each server so the services can bind on IP addresses they don't currently have configured. This allows faster failover of the services.
 
-On all your servers, add the following line in `/etc/sysctl.conf` and then reload with 'sysctl -p'
+On *all your servers*, add the following line in `/etc/sysctl.conf` and then reload with 'sysctl -p'
 
   net.ipv4.ip_nonlocal_bind = 1
 
 Then, disable the default MariaDB systemd service unit as PacketFence already provides one (packetfence-mariadb)
 
   # systemctl disable mariadb
 
-Now, on the first server, create the PEM that combines the key and certificate for the http services. Adapt to your own paths if you are using different certificates. What matters is that the path to the PEM file should be /usr/local/pf/conf/ssl/server.pem since HAProxy will expect it to be there.
+Now, on *the first server*, create the PEM that combines the key and certificate for the http services. Adapt to your own paths if you are using different certificates. What matters is that the path to the PEM file should be /usr/local/pf/conf/ssl/server.pem since HAProxy will expect it to be there.
 
   # cd /usr/local/pf/conf/ssl
   # cat server.key server.crt > server.pem
 
-Going through the configurator
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Basic PacketFence configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Now, on the first server of your cluster, you should go through the configurator, configuring the server as if it was in standalone (no HA interfaces). You should leave the services stopped at the end of the configurator.
+Now, on *the first server* of your cluster, you should go through the configurator, until *step 6* (Fingerbank configuration). You should leave the services stopped at the end of the configurator.
 
-Then restart PacketFence's mariadb on the first server:
+Then restart PacketFence's mariadb on *the first server*:
 
   # systemctl restart packetfence-mariadb
 
-On the other servers of your cluster, configure only the network interfaces without going past that section in the configurator. If the other servers already have the right IP addresses configured on their interfaces, you can ignore this step.
+On the *other servers* of your cluster, configure only the network interfaces (step 2) without going past that section in the configurator. If the other servers already have the right IP addresses configured on their interfaces, you can ignore this step.
 
-Step 3 : Create a new cluster
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+At this point, for a VLAN enforcement configuration for example, the network interfaces of your servers must be configured, and you must be able to see, for each server:
+
+|===
```

Comment:
If the guy went through the configurator, these interfaces are already there. Not sure why they should check this and IMO this section isn't very clear

Reply:
Yes, it's just a check to confort the user that thoses steps are ok (To show them as well what the configurator has done)

Reply:
Not really clear to me the way its written.

And really not sure this belongs in this guide. To me its not really useful to tell the user this happened in the clustering guide. 

Maybe that could go in a section explaining how the configurator works (in cluster or not, since it applies to both). Or in a troubleshooting section the user can use in the clustering guide maybe?

Basically, I'd just leave that out of the simple setup.